app/uk/gov/hmrc/cataloguefrontend/whatsrunningwhere/ReleasesConnector.scala

```diff
@@ -62,7 +60,19 @@ class ReleasesConnector @Inject()(http: HttpClient,
 
   def ecsReleases(profile: Option[Profile])(implicit hc: HeaderCarrier): Future[Seq[ServiceDeployment]] = {
     val baseUrl = s"$serviceUrl/releases-api/ecs-deployment-events"
-    val params = profileQueryParams(profile)
+    val params  = profileQueryParams(profile)
+    http
+      .GET[Seq[ServiceDeployment]](baseUrl, params)
+      .recover {
+        case NonFatal(ex) =>
+          Logger.error(s"An error occurred when connecting to $baseUrl: ${ex.getMessage}", ex)
+          Seq.empty
+      }
+  }
+
+  def heritageReleases(profile: Option[Profile])(implicit hc: HeaderCarrier): Future[Seq[ServiceDeployment]] = {
```

Comment:
Since this is the same as ecsReleases apart from the url, is it worth creating an internal one with the url as a param?

Reply:
turns out this call wasn't used so i'll remove it